typescript/type_literal.go

```diff
@@ -33,13 +33,17 @@ type PropertySig struct {
 }
 
 func (ps *PropertySig) WriteCode(writer CodeWriter) {
+	ps.writeIdAndTypeAnnotation(writer)
```

Comment:
whats this change?

Reply:
It was some intermediate code that I was using that i forgot to cleanup. its gone now.